EIPS/eip-7532.md

```diff
@@ -0,0 +1,44 @@
+---
+eip: 7532
+title: Generation unit (64 epochs)
+description: A Generation is a scale version of an epoch.
```

Comment:
You have a bit more room in the description field. Would you mind expanding a bit?